packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -168,6 +174,17 @@ const EntityLayoutWrapper = (props: { children?: ReactNode }) => {
   );
 };
 
+const techdocsContent = (
+  <EntityLayout.Route path="/docs" title="Docs">
```

Comment:
This isn't quite at the same level as the other content variables, perhaps we could keep the top-level route elements where they are?

Reply:
Agree, done ✔️